src/common/common.cc

```diff
@@ -29,6 +30,23 @@ UniqueID globally_unique_id(void) {
   return result;
 }
 
+/* ObjectID hashing function. */
+size_t hashObjectID(const ObjectID &key) {
```

Comment:
Now we have two separate different ID hashing functions (the other is defined in `plasma_store.h`), we should probably just define it once.